### PR TITLE
More Error Handling on New Trip

### PR DESCRIPTION
**Pull request by ivanderjmw**
Allow exit on first run. Prevent user from creating duplicate trips.

### CODE REVIEW

**Comment by ivanderjmw** [comment]
Fix #79

**Review by weishuangtan — APPROVED** [review]
lgtm!

File: src/main/java/seedu/trippie/command/NewTripCommand.java [hunk 1]
```diff
@@ -17,11 +17,18 @@
     private boolean isFilenameValid(String file) {
         Pattern p = Pattern.compile("[<>:\"/\\\\|?*]");
 
-        if (p.matcher(file).find()) {
-            return false;
-        } else {
-            return true;
+        return !p.matcher(file).find();
```
Comment (ivanderjmw) [inline src/main/java/seedu/trippie/command/NewTripCommand.java:20]: Simplified the if-else statement.

File: src/main/java/seedu/trippie/command/NewTripCommand.java [hunk 2]
```diff
@@ -17,11 +17,18 @@
...
+    }
+
+    // Implemented with reference to
+    // https://www.rgagnon.com/javadetails/java-check-if-a-filename-is-valid.html
+    private boolean doesTripNameExist(TrippieData trippieData, String name) {
+        for (Trip trip: trippieData.getTripList()) {
+            if (trip.getName().equals(name)) {
+                return true;
+            }
```
Comment (ivanderjmw) [inline src/main/java/seedu/trippie/command/NewTripCommand.java:29]: Simple O(n) linear search, should not affect performance.

**Pull request merged**